apps/app/src/features/openai/client/services/knowledge-assistant.tsx

```diff
@@ -231,12 +236,37 @@ export const useKnowledgeAssistant: UseKnowledgeAssistant = () => {
     );
   }, [dropdownOpen, toggleDropdown, form, t]);
 
+  const threadTitleView = useMemo(() => {
+    const { threadData } = aiAssistantSidebarData ?? {};
+
+    if (threadData?.title == null) {
+      return null;
```

Comment:
以下のようにすると、戻り値が JSX.Element に絞れてシンプルになると思います。
```suggestion
      return <></>;
```

AiAssistantSidebar.tsx でも null check が不要になります。